packages/cli/test/dev/utils.js

```diff
@@ -522,7 +522,7 @@ async function ps(parentPid, pids = {}) {
       : ['ps', '-o', 'pid', '--no-headers', '--ppid', parentPid];
 
   try {
-    const { stdout: buf } = spawnSync(cmd[0], cmd.slice(1), {
+    const { stdout: buf } = execFileSync(cmd[0], cmd.slice(1), {
```

Comment:
**requirement (blocking):** support different return value

`execFileSync` has a different return value than `spawnSync`. The response is stdout, not an object containing a `stdout` key.

```
> spawnSync('echo', ['hey', 'there']).stdout.toString()
'hey there\n'

> execFileSync('echo', ['hey', 'there']).toString()
'hey there\n'
```


I think this resolves it:

```suggestion
    const buf = execFileSync(cmd[0], cmd.slice(1), {
```

Reply:
Good catch!